File: pset6/mario/less/mario.py
from cs50 import get_int
def main():
	while True:

		height = get_int("Height: ")
		x =height
		if height >= 0 and height<=23:
			break
	for i in range(1, height + 1):
		hashes = i
		spaces = x - hashes
		print(" " * spaces, end="")
		print("#" * hashes, end="")

		print("  ", end="")

		print("#" * hashes)
if __name__ == "__main__":
	main()

File: pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

void repeat(char c, int times)
{
    while (--times >= 0)
    {
        printf("%c", c);
    }
}

int main()
{
    int height, width;

    do
    {
        height = get_int("Height: ");
        width = height;
    }
    while (height < 0 || height > 23);

    for (int i = 1; i <= height; i++)
    {
        int num_hashes = i;
        int num_spaces = width - num_hashes;

        repeat(' ', num_spaces);
        repeat('#', num_hashes);

        printf("  ");

        repeat('#', num_hashes);

        printf("\n");
    }
}